tools/build.ps1

```diff
@@ -71,3 +71,12 @@ if ($DevKit) {
 if ($RuntimeKit) {
     tools/create-runtime-kit.ps1 -Config $Config
 }
+
+# Build the MSI installer
```

Comment:
Generally, the installer must be build after signing. And since signing is a separate script/step I think it's best to just have a separate ps1 for this.

Reply:
Agreed. I'm happy to split that out after the fact, though.

Reply:
Right, this is why I positioned it in that position, so that it'll package the signed binaries (see also mods in `sign.ps1`).